### PR TITLE
Implement strikethrough and monospace formatting

### LINKED ISSUE #1754: Implement strikethrough formatting (0x1E)
https://modern.ircdocs.horse/formatting.html#strikethrough

### LINKED ISSUE #1755: Implement monospace formatting (0x11)
https://modern.ircdocs.horse/formatting.html#monospace

### PR DESCRIPTION
**Pull request by grissly-man**
Resolves #1754.
Resolves #1755.

### CODE REVIEW

File: client/js/keybinds.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ const colorsHotkeys = {
 	u: "\x1F",
 	i: "\x1D",
 	o: "\x0F",
+	"-": "\x1e",
```
Comment (astorije) [inline client/js/keybinds.js]: This shortcut is used to zoom out, and I think zooming in/out is rather useful in The Lounge. What about using "s"? It's unlikely that someone ever needs to "save" the page IMO. @xPaw, thoughts?

Comment (xPaw) [reply client/js/keybinds.js]: Agreed.

File: test/client/js/libs/handlebars/ircmessageparser/parseStyle.js [hunk 2]
```diff
@@ -48,6 +50,29 @@ describe("parseStyle", () => {
 		expect(actual).to.deep.equal(expected);
 	});
 
+	it("should parse strikethrough", () => {
```
Comment (xPaw) [inline test/client/js/libs/handlebars/ircmessageparser/parseStyle.js]: Can you add a couple more tests for this, like combining with italics, colors and stuff like that.

File: client/js/keybinds.js (hunk 1 above)
Comment (grissly-man) [reply client/js/keybinds.js]: Yes good point - updated

File: test/client/js/libs/handlebars/ircmessageparser/parseStyle.js (hunk 2 above)
Comment (grissly-man) [reply test/client/js/libs/handlebars/ircmessageparser/parseStyle.js]: Absolutely

**Comment by xPaw** [comment]
@grissly-man thanks for this! Do you want to also implement monospace (#1755)? It should be pretty much the same way.

**Comment by grissly-man** [comment]
@xPaw Yes absolutely - I'll get on #1755 right after this

**Comment by xPaw** [comment]
Thanks for adding more tests.

> I'll get on #1755 right after this

Do you want to do this as a separate PR or part of this one?

**Comment by grissly-man** [comment]
@xPaw Whatever you'd prefer! Doesn't really make a difference to me

**Comment by xPaw** [comment]
@grissly-man Probably simpler if you do it in this PR.

**Comment by grissly-man** [comment]
@xPaw Heard that

**Comment by grissly-man** [comment]
@xPaw heard that, pushed

**Comment by xPaw** [comment]
These keybinds need to be added to autocompletion help in `client/views/windows/help.tpl`.

**Comment by astorije** [comment]
@grissly-man, I took the liberty to squash your commits into 2 meaningful commits (one for strikethrough and one for monospace), and I added the doc to the help section.

Thanks a lot for your help!!

**Pull request merged**